### ISSUE TITLE
Issues when using plugin-proposal-class-properties 

### ISSUE THREAD

**Issue by neshanthan** [dollarshaveclub/cloudworker#50]
Hey,

I wanted to try and use this for my project and I ran into an issue, it works fine in production. I am using the babel plugin-proposal-class-properties, and for the following code

```
addEventListener('fetch', event => {
    event.respondWith(fetchAndApply(event.request))
  })
 
  async function fetchAndApply(request) {
    return new Response('hello world')
  }

class HelloWord {
    static hello = new TextEncoder();
    world(text) {
        const a = "Testing";
        return text;
    }
  }
```
I will get the error "ReferenceError: TextEncoder is not defined", but it will work fine when creating an instance of new TextEncoder inside a function or the constructor.

**Comment by hankjacobs** [comment]
Hi @neshanthan, 

Why are using plugin-proposal-class-properties? Class syntax is available for use in Cloudworker (assuming you are using a version of node that supports it). Furthermore, when is that error shown?

**Comment by neshanthan** [comment]
> Class syntax

Hey,

Class syntax is supported but I don't believe Class field declarations are available without doing the babel translations (maybe in some browsers but not chrome), its still an [experimental feature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#Field_declarations). If you try the following code in chrome:

This will work fine
```
class Hello { 

  constructor() {
    this.World = "world";
  }

  HelloWorld(){ 
    console.log("Hello" + this.World );
  } 

}
```

but this will not

```
class Hello { 

  static World = "world";

  HelloWorld(){ 
    console.log("Hello" + Hello.World );
  } 

}
```

The exact error shown by cloudworker is as follows:
![image](https://user-images.githubusercontent.com/13458675/51644435-0efc8200-1f68-11e9-80c5-8a9e140553a6.png)

which is caused by the babel translated line

`_defineProperty(HelloWord, "hello", new TextEncoder());
`
and if I try and run it directly without using babel, I will get the normal token error:
![image](https://user-images.githubusercontent.com/13458675/51644537-61d63980-1f68-11e9-9c68-c266e80cbf40.png)

**Comment by hankjacobs** [comment]
Ah, I see. Do you mind sharing file generated by babel as well?

**Comment by hankjacobs** [comment]
Thanks for your patience @neshanthan. So the Encoding APIs are only available in the [cloudworker v0.0.10 beta](https://github.com/dollarshaveclub/cloudworker/releases/tag/v0.0.10-beta.3) at the moment. I tested using the code you provided and the most recent beta and it worked for me. You can install the beta using:

```
npm install -g @dollarshaveclub/cloudworker@beta
```

Feel free to comment if that doesn't resolve the issue. In the mean time, I'm closing this issue. Thanks.
